src/components/charts/modal/DeployChart.scss

```diff
@@ -1,6 +1,6 @@
 .deploy-chart-container {
     display: grid;
-    grid-template-rows: 60px 1fr 64px;
```

Comment:
Is this intentional?

Reply:
yes...because here we need to remove the header's part